app/javascript/dashboard/routes/dashboard/settings/canned/AddCanned.vue

```diff
@@ -125,3 +131,19 @@ export default {
   },
 };
 </script>
+
+<style scoped lang="scss">
+::v-deep .mention--box {
+  border-left: 1px solid var(--color-border);
+  border-right: 1px solid var(--color-border);
+  left: 0;
+  margin: auto;
+  right: 0;
+  top: 10rem !important;
+  width: 90%;
+}
+
+::v-deep .ProseMirror-menubar {
+  display: none;
+}
```

Comment:
Instead of using v-deep as separate for different classes, I think we can use it this way.

```suggestion
::v-deep {
  .mention--box {
    border-left: 1px solid var(--color-border);
    border-right: 1px solid var(--color-border);
    left: 0;
    margin: auto;
    right: 0;
    top: 10rem !important;
    width: 90%;
  }

  .ProseMirror-menubar {
    display: none;
  }
}
```